src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -526,6 +526,7 @@ function make_compute_exp_tendency(model::PlantHydraulicsModel, canopy)
         fa_roots = p.canopy.hydraulics.fa_roots
         FT = eltype(t)
         @inbounds for i in 1:(n_stem + n_leaf)
+            im1 = i - 1
```

Comment:
I think it might be more clear to just use `fa.:($$(i - 1))` here instead of `im1` (like with `ip1` above), but either is fine :)

Reply:
will change this here too

Reply:
changing back...

Reply:
Either way, let's address the confusion by opening an issue for documenting how this interpolated `getproperty` works in a more central place (since it's common to multiple repos)

Reply:
ok! That would be great. Ill open an issue in ClimaCore unless there is a better place?